plugins/toolkit/jetbrains-ultimate/src-253+/software/aws/toolkits/jetbrains/remoteDev/caws/CodeCatalystGatewayClientCustomizer.kt

```diff
@@ -0,0 +1,25 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.remoteDev.caws
+
+// TODO: GatewayClientCustomizationProvider removed in 2025.3 - investigate new Gateway customization APIs
+/*
+import com.intellij.openapi.extensions.ExtensionNotApplicableException
+import com.jetbrains.rdserver.unattendedHost.customization.controlCenter.GatewayClientCustomizationProvider
```

Comment:
enable `com.jetbrains.codeWithMe`
and use `com.jetbrains.rdserver.unattendedHost.customization.GatewayClientCustomizationProvider`